### DISCUSSION TITLE
How to highlight search term in summary of document

### DISCUSSION THREAD

**Discussion by nicolesherman00** [microsoft-search/pnp-modern-search#2902] [General]
Is there any way to make it so the hithighlightsummary is instead showing the blurb of the document that includes your search term instead of something random?

![image](https://user-images.githubusercontent.com/22782498/231294666-1d7a3099-1bc3-4a83-bc60-afca16dbf30f.png)

Here I searched for term "Sliding fee"
but for the first 3 documents it is not obvious that the term is even included in the documents.

**Comment by kasperbolarsen** [comment]
We have a number of questions about Highlighted content, like https://github.com/microsoft-search/pnp-modern-search/discussions/2746 , does that answer your question ?

**Reply by nicolesherman00** [reply]
not exactly but I did ask that user if they figured out the issue as well. Does not sound like that discussion you linked was resolved. Apologies for the duplicate post.

**Reply by wobba** [reply]
It should return something useful, if not I'd say it's an API issue. How does it look in classic or modern OOB search experiences?
